development-docs/DEV_GUIDE.md

```diff
@@ -39,6 +39,8 @@ The `make` build is using GNU versions of `find` and `sed` utilities and is not
 
 The build requires `bash` version 4+ which is not shipped Mac OS but can be installed via homebrew. You can run `brew install bash` to install a compatible version of `bash`. If you wish to change the default shell to the updated bash run `sudo bash -c 'echo /usr/local/bin/bash >> /etc/shells'` and `chsh -s /usr/local/bin/bash`
 
+Additional MacOS setup instructions can be seen in the [DEV_QUICK_START_MACOS](https://github.com/strimzi/strimzi-kafka-operator/blob/master/DEV_QUICK_START_MACOS.md) file.
```

Comment:
isn'tt this path now wrong? e.g. `[DEV_QUICK_START_MACOS](https://github.com/strimzi/strimzi-kafka-operator/blob/master/development-docs/DEV_QUICK_START_MACOS.md)`

Reply:
Also can't we just supply a relative path?
`development-docs/DEV_QUICK_START_MACOS.md`